FeedSX/models/addPostMetaData.ts

```diff
@@ -0,0 +1,20 @@
+// selected image/video format interface
+export interface ImageVideoMetaData {
+  fileName: string;
+  fileSize: number;
+  height: number;
+  type: string;
+  uri: string;
+  width: number;
+  bitrate?: number;
+  duration?: number;
+  originalPath?: string;
+}
+// selected document format interface
+export interface DocumentMetaData {
+  name: string;
+  size: number;
+  fileCopyUri: null;
```

Comment:
why it is null?

Reply:
getting null from the response of the selected file